README.md

```diff
@@ -63,174 +33,119 @@ For more information on the importance of a professional README for your reposit
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+# 📖 [ React Capstone ] <a name="about-project"></a>
+The React Capstone project, driven by a user-centric approach, involves the creation of a mobile web application. This application empowers users to efficiently track a meticulously curated set of numeric metrics by harnessing the capabilities of React, Redux, and integrating data through an API, ensuring seamless functionality and real-time data access.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
 <details>
   <summary>Client</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://javascript.com/">JavaScript</a></li>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://ReactJs.com/">ReactJs</a></li>
+    <li><a href="https://Redux.com/">Redux</a></li>
   </ul>
 </details>
 
 <details>
   <summary>Server</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="#">N/A</a></li>
   </ul>
 </details>
 
 <details>
 <summary>Database</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="#">N/A</a></li>
   </ul>
 </details>
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- Deploying the project.
+- Adding  Professional UI.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+- [Live Demo Link](https://mumbereh.github.io/react_redux_capstone/)
```

Comment:
- [ ] @mumbereh Great work on your readme, but as the previous reviewer noted the page doesn't work it's currently displaying the readme file as shown below.
![image](https://github.com/mumbereh/react_redux_capstone/assets/61408860/38e52dd6-1922-4681-908d-4b19e7e47537)